lib/modules/nightMode.js

```diff
@@ -297,16 +297,24 @@ async function getNightModeTimes() {
 			} catch (err) {
 				console.warn('Failed to init automatic night mode:', err);
 
-				return Alert.open(i18n('nightModeAutomaticNightModeDenied', 'confirm'), { cancelable: true })
-					.then(() => {
-						// they clicked confirm, turn it off
-						Options.set(module, 'automaticNightMode', 'none');
-
-						return {
-							startingTime: new Date(0),
-							endingTime: new Date(0),
-						};
-					});
+				switch (err.code) {
+					case err.PERMISSION_DENIED:
+						return Alert.open(i18n('nightModeAutomaticNightModeDenied', 'confirm'), { cancelable: true })
+							.then(() => {
+								// they clicked confirm, turn it off
+								Options.set(module, 'automaticNightMode', 'none');
+
+								return {
+									startingTime: new Date(0),
+									endingTime: new Date(0),
+								};
+							});
+					case err.POSITION_UNAVAILABLE:
+					case err.TIMEOUT:
+					case err.UNKNOWN_ERROR:
+					default:
+						return null;
```

Comment:
This function can't return `null` or it'll cause errors elsewhere, see the [Flow output in CI](https://travis-ci.org/honestbleeps/Reddit-Enhancement-Suite/jobs/291191270). We can probably just `throw err;` in this case.

Reply:
Done, Sir! Apologies for the delay.